pom.xml

```diff
@@ -207,8 +207,9 @@
             <snapshots>
                 <enabled>false</enabled>
             </snapshots>
-            <id>geowave</id>
-            <url>http://geowave-maven.s3-website-us-east-1.amazonaws.com/release</url>
+            <id>osgeo-release</id>
+            <name>Open Source Geospatial Foundation Repository</name>
+            <url>https://repo.osgeo.org/repository/release/</url>
         </repository>
```

Comment:
These artifacts are available in maven central, so we should remove this entire repository block and pull from central instead.

Reply:
I don't think the boundless artifacts are available in maven central.  at least, it wouldn't build for me without this repo.

Reply:
Wait, so they've deployed artifacts to maven central that have dependencies that aren't in maven central?! That seems like a bug of theirs to me. Sigh.